### PR TITLE
Redirect A File Icon package

### PR DESCRIPTION
**Pull request by deathaxe**
Redirect the A File Icon package to a reconstructed repository to re-up its functionality.

### CODE REVIEW

**Comment by deathaxe** [comment]
Please wait with merging. The repo name may change once in the near future.

**Comment by deathaxe** [comment]
Done.

**Comment by mjoe** [comment]
I've already looked for a successor of this package. @wbond Thank you for merging this PR soon.

File: repository/a.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
 		{
 			"name": "A File Icon",
 			"previous_names": ["File Icons Extended", "zz File Icons"],
-			"details": "https://github.com/ihodev/a-file-icon",
-			"author": "ihodev",
+			"details": "https://github.com/SublimeText/AFileIcon",
+			"author": "ihodev, DeathAxe",
```
Comment (FichteFoll) [inline repository/a.json]: This should have been a list of authors.

**Comment by deathaxe** [comment]
Already fixed by https://github.com/wbond/package_control_channel/pull/7424

**Pull request merged**